src/panel/direction/RouteVia.jsx

```diff
@@ -5,20 +5,16 @@ import PublicTransportLine from './PublicTransportLine';
 const RouteVia = ({ route, vehicle }) => {
   if (vehicle !== 'publicTransport') {
     return <div className="routeVia">
-      {_('Via', 'direction')} {route.summary.replace(/^(.*), (.*)$/, '$1')}
+      {_('Via', 'direction')} {route.legs[0].summary.replace(/^(.*), (.*)$/, '$1')}
     </div>;
   }
 
   return <div className="routeVia">
-    {route.steps
-      // removes multiple 'WALK' parts
```

Comment:
This data sanitization hack is not needed anymore. Public transport routes now include a `summary` array with info about each leg.